src/EditorFeatures/CSharpTest/CodeActions/AddAwait/AddAwaitTests.cs

```diff
@@ -368,6 +366,1086 @@ async void M()
     {
         var x = (await GetNumberAsync()).Invalid();
     }
+}");
```

Comment:
pulled all these in from  src/EditorFeatures/CSharpTest/Diagnostics/Async/AddAwaitTests.cs (which was deleted)

Reply:
Question, is it important for the existing tests in this file to also have the `CodeActionsAddAwait` trait like the tests that were carried over? Is the trait just for discoverability purposes?